Resources/Prototypes/Entities/Mobs/Player/familiars.yml

```
@@ -148,6 +150,8 @@
     name: ghost-role-information-cerberus-name
     description: ghost-role-information-cerberus-description
     rules: ghost-role-information-cerberus-rules
+    mindRoles:
+    - MindRoleGhostRoleFamiliar
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)